### ISSUE TITLE
Feature idea: Permanent prompt pre/su-ffix.

### ISSUE THREAD

**Issue by arthurwolf** [backnotprop/prompt-tower#7]
TLDR: It'd be nice to have a configurable block of text that's always put before/after the prompt.

ChatGPT offers a system where you can tell it "things about yourself" or things about "how you want your prompt handled" etc.

I give it my preferences like the libraries I want it to use etc, the issue is the box is pretty short and I've already filled it.

It'd be neat if prompt-tower would let me configure some text that is always put at the beginning of the prompt (and I guess while you're at it, maybe somebody else needs text at the end too?).
 
That'd let me give it all my preferences and be sure they are always a part of the prompts I make/send.

**Comment by backnotprop** [comment]
#34 will address
